diff --git a/node_modules/@react-native/babel-preset/src/configs/main.js b/node_modules/@react-native/babel-preset/src/configs/main.js
index 077e1a6..d498cd1 100644
--- a/node_modules/@react-native/babel-preset/src/configs/main.js
+++ b/node_modules/@react-native/babel-preset/src/configs/main.js
@@ -27,7 +27,7 @@ const loose = true;
 const defaultPlugins = [
   [require('babel-plugin-syntax-hermes-parser'), {parseLangTypes: 'flow'}],
   [require('babel-plugin-transform-flow-enums')],
-  [require('@babel/plugin-transform-block-scoping')],
+  // [require('@babel/plugin-transform-block-scoping')],
   [require('@babel/plugin-transform-class-properties'), {loose}],
   [require('@babel/plugin-transform-private-methods'), {loose}],
   [require('@babel/plugin-transform-private-property-in-object'), {loose}],
@@ -88,7 +88,7 @@ const getPreset = (src, options) => {
   }
 
   if (!isHermes && (isNull || src.includes('=>'))) {
-    extraPlugins.push([require('@babel/plugin-transform-arrow-functions')]);
+    // extraPlugins.push([require('@babel/plugin-transform-arrow-functions')]);
   }
 
   if (!isHermes) {
@@ -130,10 +130,10 @@ const getPreset = (src, options) => {
     );
   }
   if (isNull || src.indexOf('async') !== -1) {
-    extraPlugins.push([
-      require('@babel/plugin-transform-async-generator-functions'),
-    ]);
-    extraPlugins.push([require('@babel/plugin-transform-async-to-generator')]);
+    // extraPlugins.push([
+    //   require('@babel/plugin-transform-async-generator-functions'),
+    // ]);
+    // extraPlugins.push([require('@babel/plugin-transform-async-to-generator')]);
   }
   if (
     isNull ||
